webcompat/templates/layout.html

```diff
@@ -39,14 +39,14 @@
   {% include "shared/footer.html" %}
 </div>
 {%- if config.PRODUCTION or config.STAGING -%}
+<!-- Google Analytics -->
 <script>
-  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
-  ga('create', 'UA-49507820-1', 'webcompat.com');
-  ga('send', 'pageview');
+window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
```

Comment:
Can we create a new script file and link to it here, rather than have inline script? That way we can remove the need to allow data URIs via CSP.

It could live at `webcompat/static/js/vendor/ga.js`.